src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -218,13 +219,13 @@
             new Object[] {
                 "IndirectlyRelatedPairs",
                 "OCC",
-                0.6666666666666666d,
+                0.6_666_666_666_666_666d,
             },
             new Object[] {"MethodMethodCalls", "OCC", 0.2d},
             new Object[] {
                 "MethodsWithDiffParamTypes",
                 "OCC",
-                0.3333333333333333d,
+                0.3_333_333_333_333_333d,
```

Comment:
@GnusinPavel This seems strange to me, we don't have sepparators in the decimal part of a number. I'd prefer these to stay as before, if it doesn't fail any qulice rule.

Reply:
yes, it violates qulice rule. Ok, I'll remove underscores and put suppression